portal/settings.py

```diff
@@ -1,39 +1,138 @@
-"""
-Django settings for portal project.
+# import os 
```

Comment:
These changes are unrelated, have you tried to run the project after doing these changes ? Please try to run the project after the changes and check if it is running or not ?